playbooks/frontend.yml

```diff
@@ -25,6 +25,8 @@
         - perl-XML-LibXML
         - unzip
         - curl
+        - awslogs
+        - jq
```

Comment:
what is jq used for?

Reply:
This is a JSON parser.
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_cloudwatch_logs.html